src/packet_analysis/Manager.cc

```diff
@@ -171,3 +182,67 @@ void Manager::DumpPacket(const Packet *pkt, int len)
 
 	run_state::detail::pkt_dumper->Dump(pkt);
 	}
+
+class UnknownProtocolTimerTimer final : public zeek::detail::Timer {
+public:
+	using UnknownProtocolPair = std::pair<std::string, uint32_t>;
```

Comment:
I would add some documentation here like: Represents a pair of analyzer name and protocol identifier, where the identifier was reported as unknown for the analyzer.

Reply:
Sounds perfect.